src/notifier.rs

```diff
@@ -0,0 +1,74 @@
+extern crate notify_rust;
+
+pub use tui::messaging::Timestamp;
+use tui::MsgTarget;
+
+use self::notify_rust::Notification;
+
+
+// Destktop notification handler
+pub struct Notifier {
+    // [off,mentions,messages]
+    notify_for: String,
+}
```

Comment:
Use a `pub enum Notifier { Off, Mentions, Messages }` here (maybe with a different name). You already now the `String` is one of these (parsed by the command implementation), so you can reflect this fact in the type and avoid string comparisons below.